angular/projects/common/src/app/utils/record-constants.ts

```diff
@@ -232,7 +232,8 @@ export class Picklists {
     'nris-epd',
     'nro-inspections-csv',
     'nrpti',
-    'ocers-csv'
+    'ocers-csv',
+    'agri-mis'
```

Comment:
nitpick, put `agri-mis` at the top so the list is alphabetically sorted on the UI. Otherwise it looks great!

Reply:
👍